DataCollector/RulerDataCollector.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Hoathis\Bundle\RulerBundle\DataCollector;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+
+use Hoathis\Bundle\RulerBundle\Ruler\LoggedRuler;
```

Comment:
With @jubianchi and others, we used to use `Hoathis\Symfony${library}Bundle` pattern when creating a bundle with Hoa. So in this case, it would be: `Hoathis\SymfonyRulerBundle`. Thought?